ports/servoshell/Cargo.toml

```diff
@@ -91,6 +92,9 @@ napi-derive-ohos = "1.0.1"
 napi-ohos = "1.0.1"
 ohos-sys = { version = "0.4.0", features = ["xcomponent"] }
 ohos-vsync = "0.1.2"
+ohos-ime = { version = "0.2" }
+ohos-ime-sys = "0.1.1"
+xcomponent-sys = { version = "0.1.1", features = ["api-12", "keyboard-types"] }
```

Comment:
Does the `api-12` here imply we need to build against the OHOS 5.0 SDK?

Reply:
Building works fine with the OH 4.1 SDK, but at runtime OH 5.0 is required. The whole IME interface for native code was added in API-12.

Reply:
I get this error when building this branch with SDK version 4.1.7.5:

```
  = note: ld.lld: error: unable to find library -lohinputmethod
          clang-15: error: linker command failed with exit code 1 (use -v to see invocation)
```

Maybe I'm missing something in the configuration?

Reply:
Huh, I could have sworn I was still using the 4.1 SDK on my mac. I guess then we need to bump the SDK version in CI. It seems the book doesn't mention the minimum supported API version right now, but probably we should mention that too.

Reply:
I bumped the version to 5.0 in the arkts project and CI

Reply:
> It seems the book doesn't mention the minimum supported API version right now, but probably we should mention that too.

Yep, the book needs to be updated. We also enforce that in mach, so we should update [this](https://github.com/servo/servo/blob/aa7116c75d2f13f554ce4db162344aaea59e60b0/python/servo/platform/build_target.py#L278) in this PR.

Reply:
Thanks! I also updated the check in `mach` and upgraded it from a warning to an error if the SDK is too old.